calculator.c

```diff
@@ -3,10 +3,23 @@
 int  main(){
     int a;
     int b;
+    int div(int a , int b);
+
     printf("\nEnter any number");
     scanf("%d",&a);
     printf("\nEnter another number");
     scanf("%d",&b);
+    division(a,b);
     
     return 0;
-}
\ No newline at end of file
+
+}
+
+int division(int a,int b)
+{
+int div;
+div=a/b;
+printf("\n The result is: %d",div);
```

Comment:
loss of data due to explicit typecasting . The fraction part of result is lost due to variable div being integer type.